### PR TITLE
Next Reads: Design feedback

### PR DESCRIPTION
**Pull request by imranolas**
- [x] Update component copy/wording
- [x] Display 3 items for read next
- [x] Fix styling for header and bottom next read components
- [x] Hide all more ons when displaying next reads

### CODE REVIEW

File: client/components/read-next/main.scss [hunk 1]
```diff
@@ -38,15 +62,38 @@
...
-	&::before {
-		content: '>';
-		speak: none;
-		position: absolute;
-		top: -2px;
-		left: -15px;
+	display: inline-block;
+
+	.next-up__header & {
+		font-size: 24px;
```
Comment (i-like-robots) [inline client/components/read-next/main.scss]: Is it OK to stick with the type scale (size 2)? If it doesn't match the design that should be OK, we'll be looking at the scale again soon.

File: client/components/read-next/main.scss [hunk 2]
```diff
@@ -38,15 +62,38 @@
...
+			@include nextIcon(arrow-right, getColor('cold-2'), 17);
+			content: '';
+			speak: none;
+			position: absolute;
+			top: 8px;
+			left: -22px;
+		}
+
+		@include oGridRespondTo(M) {
+			font-size: 30px;
```
Comment (i-like-robots) [inline client/components/read-next/main.scss]: Same as above

**Comment by imranolas** [comment]
screen shot 2015-09-28 at 16 06 27
screen shot 2015-09-28 at 16 06 45

File: client/components/read-next/main.scss [hunk 3]
```diff
@@ -55,6 +102,7 @@
 	color: oColorsGetColorFor(timestamp, text);
 	text-transform: uppercase;
 	white-space: nowrap;
+	display: none;
```
Comment (i-like-robots) [inline client/components/read-next/main.scss]: Is there a need for the other styles in this block?

File: client/components/suggested-reads/main.js [hunk 4]
```diff
@@ -3,7 +3,8 @@ var fetchres = require('fetchres');
 
 module.exports = {
 	init: function(flags) {
-		if (!flags.get('articleSuggestedRead')) { return; }
+		return;
```
Comment (i-like-robots) [inline client/components/suggested-reads/main.js]: Remove the dependency and init call?

File: server/controllers/article.js [hunk 5]
```diff
@@ -90,7 +91,14 @@ module.exports = function(req, res, next) {
 					standFirst: article[0] ? article[0].item.editorial.standFirst : ""
 				}),
 				socialMediaImage(article[1]),
-				res.locals.flags.articleSuggestedRead && article[0] ? readNext(article[0], res.locals.flags.elasticSearchItemGet) : Promise.resolve()
+				res.locals.flags.articleSuggestedRead && article[0] ? readNext(article[0], res.locals.flags.elasticSearchItemGet) : Promise.resolve(),
+				getSuggested(article[0]).then(function(it) {
+					return api.contentLegacy({
+						uuid: (it && it.ids) || [],
+						useElasticSearch: res.locals.flags.elasticSearchItemGet,
+						type: 'Article'
```
Comment (i-like-robots) [inline server/controllers/article.js]: No type option

File: server/controllers/article.js [hunk 6]
```diff
@@ -100,6 +108,7 @@ module.exports = function(req, res, next) {
 			var article = results[1];
 			var mainImage = results[3];
 			var readNextArticle = results[4];
+			var readNextArticles = results[5].map(function(it) { return articlePodMapping(it); });
```
Comment (i-like-robots) [inline server/controllers/article.js]: fat arrow all the things =)

File: views/partials/related/suggested-reads.html [hunk 7]
```diff
@@ -1,12 +1,14 @@
...
-				<a href="/{{item.id}}" data-trackable="related">
-					<h3 class="article__suggested-reads__item__headline">{{item.title.title}}</h3>
-				</a>
-			</div>
-		</article>
-	{{/each}}
-</div>
+{{#if items.length}}
+	<div class="article__suggested-reads__article-list" role="group" aria-labelledBy="article__suggested-reads__title" >
+		<span id="article__suggested-reads__title" class="article__suggested-reads__title">Recommended:</span>
```
Comment (i-like-robots) [inline views/partials/related/suggested-reads.html]: Should this be a heading (h3 like the read next)?

File: client/components/read-next/main.scss (hunk 3 above)
Comment (imranolas) [inline client/components/read-next/main.scss]: Wasn't sure whether this element is expected to be in the final designs but wanted to keep the styles incase. Happy to remove completely in the meantime as we can always retrieve from history.

File: client/components/suggested-reads/main.js (hunk 4 above)
Comment (imranolas) [inline client/components/suggested-reads/main.js]: Will do.

File: client/components/read-next/main.scss (hunk 1 above)
Comment (imranolas) [inline client/components/read-next/main.scss]: I'm apprehensive to stray from the designs as they'll look pretty bad in this case. I'll mark them with a comment to be revisited upon type scale revisions.

**Pull request merged**